src/test/java/uk/nhs/hee/tis/trainee/forms/service/FormFieldValidationServicePartBTest.java

```diff
@@ -122,19 +122,31 @@ void whenLocalOfficeNameIsInvalidThenThrowsException(String str) {
   }
 
   @Test
-  void whenCurrRevalDateIsPastThenThrowsException() {
+  void whenCurrRevalDateIsNullThenThrowsException() {
     FormRPartBDto input = validForm();
-    input.setCurrRevalDate(LocalDate.MIN);
+    input.setCurrRevalDate(null);
 
     assertThrows(ValidationException.class, () -> service.validateFormRPartB(input));
   }
 
   @Test
-  void whenCurrRevalDateIsTooBigThenThrowsException() {
+  void whenCurrRevalDateIsPastThenDoesNotThrowException() {
+    FormRPartBDto input = validForm();
+    input.setCurrRevalDate(LocalDate.MIN);
+
+    service.validateFormRPartB(input);
```

Comment:
You could use `assertDoesNotThrow()` to make the intention clearer

Reply:
Good point - I just lazily copied what was already there 😪